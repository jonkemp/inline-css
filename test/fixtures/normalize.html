<html>
  <head><link rel="stylesheet" href="normalize.css" />
  </head>
  <body>
    <h1>normalize.css</h1></body></html>
