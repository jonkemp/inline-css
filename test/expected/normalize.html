<html style="-ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%; font-family: sans-serif;">
  <head>
  </head>
  <body style="margin: 0;">
    <h1 style="font-size: 2em; margin: 0.67em 0;">normalize.css</h1>
  </body>
</html>
